docs/UserGuide.md

```diff
@@ -140,7 +146,22 @@ Examples:
 
 - `fav 1` — Adds contact at index 1 to your list of favourites
 
-### Adding high importance flag: `impt`
+![favourite](images/favourite_command.png)
+
+- `fav 1` - Run the command for the same person, and the favourite status will be toggled off.
```

Comment:
Should we be using `contacts` instead of `person` over here?

Reply:
Thanks!